package.json

```diff
@@ -2,9 +2,14 @@
   "name": "eslint-config-brightspace",
   "version": "1.3.1",
   "description": "Common Brightspace eslint configs.",
-  "main": "index.js",
+  "exports": {
+    ".": "./index.js",
+    "./utils.js": "./utils.js"
```

Comment:
I'd be tempted to just export these utilities from `index.js` so that consumer's imports can all be from one place.